app/build.gradle

```diff
@@ -84,8 +87,8 @@ android {
     }
     variantFilter { variant ->
         def names = variant.flavors*.name
-        if (names.contains("fdroid") && variant.buildType.name == "debug") {
-            // We only need the F-Droid release version
+        if ((names.contains("fdroid") || names.contains("internal")) && variant.buildType.name == "debug") {
+            // We only need release version for F-Droid and internal builds
```

Comment:
will add the same comment here as from the other PR; I think this phrasing could be clearer

Reply:
you mean comment in line 91? I tis the same comment. Just copied it across from the other PR. Maybe I missed something?

Reply:
yeah that's clearer, thanks. Think my other suggestion was missed but it was basically what you've ended up with. 👍 

[lll
![C9856D65-D05C-49A9-BE79-6E35EC1639BC](https://user-images.githubusercontent.com/1336281/125050172-a521b580-e099-11eb-9e55-1d75759a845a.jpeg)